libkbfs/cr_chains.go

```diff
@@ -660,7 +667,7 @@ func (ccs *crChains) addOps(codec kbfscodec.Codec,
 	privateMD PrivateMetadata, winfo writerInfo,
 	localTimestamp time.Time) error {
 	// Copy the ops since CR will change them.
-	ops := make(opsList, len(privateMD.Changes.Ops))
+	var ops opsList
```

Comment:
I had the wrong idea about how codec worked, but a quick experiment told me you were right. Changed back!